.circleci/config.yml

```diff
@@ -37,6 +37,7 @@ jobs:
       - run: # Run linter after build because the integration test code depend on compiled typescript...
           name: Linter check
           command: npm run lint
+          chart_name: hmpps-historical-prisoner
```

Comment:
had to put all these in as I renamed the project but haven't renamed the deployment etc. yet